platforms/azure/giantnetes/variables.tf

```diff
@@ -159,6 +159,15 @@ variable "vnet_cidr" {
   default     = "10.0.0.0/16"
 }
 
+# NOTE:
+# - bastion_cidr should be a part of vnet_cidr.
+# - bastion_cidr should be unique across all installations if VPN enabled.
+# - recommended to use /28 subnets from range 10.0.4.0/22 (for default 10.0.0.0/16 vnet_cidr).
```

Comment:
Just for my information, did you understood where this recommendation of /28 came from ?

Reply:
It's my recommendation, so we can have this as baseline. No strict policy tho. `/28` is used in AWS.

Any requirements from BYOC side?

Reply:
For reference https://github.com/giantswarm/giantswarm/wiki/Giant-Swarm-VPN

Reply:
Ok, for info I made it /24 on guest cluster because why not? and I could not find a reason not to.

Reply:
`/24` is also fine. In guest clusters we already by design have non-intersecting networks, so we just fine with any range.

In VPN server <=> host clusters we should be careful, because we use almost always the same range for host cluster `10.0.0.0/16` That's why i'm recommending to take `10.0.4.0/22` as range and pick up `/28` networks from it. This give us 64 potential host clusters. And we always can pick up another range if this one will be occupied.